### PR TITLE
feat: login screen

### PR DESCRIPTION
**Pull request by igorosip0v**
Closes [WID-680](https://linear.app/worldcoin/issue/WID-680/onboarding-page-flow)

### Changes: 
- Realized that we don't need common layout inside (Onboarding) routes. So I've regrouped it to make separate layout for each page in this group.
- Added create-team page
  - Should be under auth0 protection when #474 will be merged
- Added common Typography component as we have standardized typography in most places
- Added `Checkbox` common component
- Added `LoggedUserNav` common component. [Figma](https://www.figma.com/file/URO9Y5UPOqK3LbLJ9623Gv/%F0%9F%AA%AA-World-ID-2.0?node-id=1837%3A9213&mode=dev)
- Added `LayersIconFrame` common component. [Figma](https://www.figma.com/file/URO9Y5UPOqK3LbLJ9623Gv/%F0%9F%AA%AA-World-ID-2.0?type=design&node-id=1837-9556&mode=dev)
- Added `SizingWrapper` component to make common `x` paddings

--- 

https://github.com/worldcoin/developer-portal/assets/89008845/046017f0-ca73-482e-b01d-5fd20386aeb8

### CODE REVIEW

File: web-app-router/scenes/Onboarding/CreateTeam/page/index.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+      <div className="grid max-w-[580px] w-full gap-y-10">
+        <LayersIconFrame>
+          <WorldcoinBlueprintIcon />
+        </LayersIconFrame>
+
+        <Typography as="h1" variant={TYPOGRAPHY.H6} className="text-center">
+          Create your first team
+        </Typography>
+
+        <Form />
```
Comment (andy-t-wang) [inline web-app-router/scenes/Onboarding/CreateTeam/page/index.tsx:18]: To be honest the form feels like it's a bit too wide on the small screen just like 15 or 20 px

**Review by andy-t-wang — APPROVED** [review]

File: web-app-router/scenes/Onboarding/Login/page/index.tsx [hunk 2]
```diff
@@ -1,5 +1,40 @@
...
+  <div className="w-full h-full flex items-center justify-center">
+    <div className="grid gap-y-10 max-w-[360px]">
+      <LayersIconFrame>
+        <WorldcoinBlueprintIcon />
+      </LayersIconFrame>
+
+      <div className="grid gap-y-3">
+        <Typography as="h1" variant={TYPOGRAPHY.H6}>
+          World ID is now generally available
+        </Typography>
```
Comment (andy-t-wang) [inline web-app-router/scenes/Onboarding/Login/page/index.tsx:16]: This text needs to be centered

File: web-app-router/scenes/Onboarding/Login/page/index.tsx [hunk 3]
```diff
@@ -1,5 +1,40 @@
...
+
+        <Typography as="p" variant={TYPOGRAPHY.R3} className="text-grey-500">
+          The Worldcoin Protocol will enable a new class of applications built
+          on top of prof of personhood
+        </Typography>
+      </div>
+
+      <div className="grid gap-y-4">
+        {/* TODO: update url */}
+        <DecoratedButton href="/create-team" className="py-3">
```
Comment (andy-t-wang) [inline web-app-router/scenes/Onboarding/Login/page/index.tsx:26]:
```suggestion
        <DecoratedButton href="/create-team" className="py-4">
```

File: web-app-router/scenes/Onboarding/Login/page/index.tsx [hunk 4]
```diff
@@ -1,5 +1,40 @@
...
+          Create an account
+        </DecoratedButton>
+
+        <DecoratedButton
+          variant="secondary"
+          className="py-3"
```
Comment (andy-t-wang) [inline web-app-router/scenes/Onboarding/Login/page/index.tsx:32]: ``
```suggestion
          className="py-4"
```

File: web-app-router/scenes/Onboarding/CreateTeam/page/index.tsx (hunk 1 above)
Comment (igorosip0v) [reply web-app-router/scenes/Onboarding/CreateTeam/page/index.tsx]: @andy-t-wang I've added `SizingWrapper` around this screen to make common paddings

**Pull request merged**